src/lib/generator/writeJson.js

```diff
@@ -44,166 +72,77 @@ function parseCsvFile(csvFile) {
                 description,
                 hcc_v24,
                 hcc_v28,
+                is_specific: true,
             };
 
-            // Check if there's a previous entry with the same code
-            const prevEntry = _.find(memo, { code });
-
-            // If a previous entry exists, merge the current data with it and push to the memo array
-            if (prevEntry) {
-                const mergedEntry = _.merge(obj, prevEntry);
-                // @ts-ignore
-                memo.push(mergedEntry);
-                return memo;
-            }
+            memo[code] = _.merge(obj, memo[code]);
```

Comment:
not entirely sure which file we want to prefer but this is as it was.. the icd text file description will win over the hcc one